modules/flye/meta.yml

```diff
@@ -0,0 +1,65 @@
+name: "flye"
+description: De novo assembler for single molecule sequencing reads
+keywords:
+  - assembly
+  - genome
+  - de novo
+  - genome assembler
+  - single molecule
+tools:
+  - "flye":
+      description: "Fast and accurate de novo assembler for single molecule sequencing reads"
+      homepage: "https://github.com/fenderglass/Flye"
+      documentation: "https://github.com/fenderglass/Flye/blob/flye/docs/USAGE.md"
+      tool_dev_url: "https://github.com/fenderglass/Flye"
+      doi: "doi:s41592-020-00971-x"
+      licence: "['BSD-3-clause']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - reads:
+      type: file
+      description: Input file in FASTA/FASTQ format.
+      pattern: "*.{fasta,fastq,fasta.gz,fastq.gz,fa,fq,fa.gz,fq.gz}"
+
```

Comment:
Missing description of `mode` input channel

Reply:
The pattern could be an or statment of the different parameters: 

```
pattern: "--pacbio-raw|--pacbio-corr|--nano-raw<...>"
```

Reply:
Shouldn't mode be part of the `$args`?

Reply:
I made it a separate variable as it's mandatory

Reply:
I also generally argue if there is a mandatory parameter that influences input/output it should be included as an input/output channel